core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1878,25 +1879,25 @@ private void journalSetAttribute(long fileId, long opTimeMs, SetAttributeOptions
    */
   public void scheduleAsyncPersistence(AlluxioURI path) throws AlluxioException {
     synchronized (mInodeTree) {
-      scheduleAsyncPersistenceInternal(path);
       long fileId = mInodeTree.getInodeByPath(path).getId();
+      scheduleAsyncPersistenceInternal(fileId);
       // write to journal
       AsyncPersistRequestEntry asyncPersistRequestEntry =
           AsyncPersistRequestEntry.newBuilder().setFileId(fileId).build();
       writeJournalEntry(
           JournalEntry.newBuilder().setAsyncPersistRequest(asyncPersistRequestEntry).build());
       flushJournal();
     }
+    mAsyncPersistHandler.scheduleAsyncPersistence(path);
   }
 
   /**
-   * @param path the path to schedule asynchronous persistence for
+   * @param fileId the id of the file to schedule asynchronous persistence for
    * @throws AlluxioException if scheduling fails
    */
-  private void scheduleAsyncPersistenceInternal(AlluxioURI path) throws AlluxioException {
-    Inode<?> inode = mInodeTree.getInodeByPath(path);
+  private void scheduleAsyncPersistenceInternal(long fileId) throws AlluxioException {
```

Comment:
I would prefer to keep the name for consistency with other methods that follow this pattern